packages/core/src/components/radio/radio.scss

```diff
@@ -0,0 +1,105 @@
+@use '../../helpers';
+@use '../../mixins';
+
+@mixin Radio() {
+  $boundary: helpers.space(6);
+  $size: helpers.space(2.75);
+  $border-width: 1px;
+
+  .ods-radio-label {
+    @include mixins.interactive();
+
+    $padding: ($boundary - $size) / 2;
+
+    display: inline-flex;
+    flex-direction: row;
+    padding: $padding;
+    position: relative;
+    transition: background-color, border-color 0.15s ease-out;
+
+    &.-bordered {
+      border: $border-width solid helpers.color('border-separator');
+      border-radius: helpers.border-radius('medium');
+      display: flex;
+      padding: $padding - $border-width;
+    }
+
+    &.-bordered.-checked {
+      background-color: helpers.color('background-action-subtle');
+      border-color: helpers.color('border-action-subtle');
+    }
+
+    &.-disabled {
+      border-color: helpers.color('border-disabled');
+    }
+
+    > :not([aria-hidden='true']) {
+      display: flex;
+      flex-direction: column;
+      padding-left: helpers.space(1.625);
+      padding-right: helpers.space(0.375);
+      white-space: pre-line;
```

Comment:
🤔  What's the intention here?

Reply:
I suspect description can be long text with new lines? @Subcide would it be a case?

Reply:
Ah I see, a downside of accepting only strings.

If we do JSX we can have `<br />` tags in there.

```tsx
<Radio>
  Label
  <Description>
    Line 1<br />
    Line 2
  </Description>
</Radio>
```

Reply:
I'm changing this to be with JSX.